extensions/chocolatey-vscode.extension/README.md

```diff
@@ -0,0 +1,57 @@
+# chocolatey-vscode.extension
+
+This is a Chocolatey extension that simplifies building Chocolatey packages which install Visual Studio Code extensions.
+
+## Installation
+
+Install via chocolatey: `choco install chocolatey-vscode.extension`.
+
+The module is usually automatically installed as a dependency.
+
+## Requirements
+
+This extension requires the `chocolatey-core.extension`.
+
+## Usage
+
+To create a package that uses a function from this extension add the following to the `nuspec` specification:
+
+```xml
+<dependencies>
+    <dependency id="chocolatey-core.extension" version="SPECIFY_LATEST_VERSION" />
```

Comment:
since this is a required dependency, would probably be better to have this package depend on it.
Instead of relying on the maintainer to specify the dependency.

Reply:
done